pages/kids/pdf/[kidsDictionaryId].tsx

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable @next/next/no-img-element */
+import React, { useMemo, FunctionComponent } from 'react';
+import { getCountryVariant } from 'utils/locales';
+import { GetStaticPaths, GetStaticProps, InferGetStaticPropsType } from 'next';
+import { getServerSideTranslations } from '../../../utils/localization';
+import { useLanguage } from '../../../utils/useLanguageHook';
+import { DictionaryDataObject, fetchDictionary, getKidsCategory, Phrase } from '../../../utils/getDataUtils';
+import { ParsedUrlQuery } from 'querystring';
+
+interface UrlParams extends ParsedUrlQuery {
+  kidsDictionaryId: string;
+}
+
+interface PhraseImageProps {
+  phrase: Phrase;
+}
+
+const KidsDictionaryPage = ({ dictionary }: InferGetStaticPropsType<typeof getStaticProps>): JSX.Element => {
+  const kidsCategory = useMemo(() => getKidsCategory(dictionary), [dictionary]);
+  const { currentLanguage, otherLanguage } = useLanguage();
+
+  const PhraseImage: FunctionComponent<PhraseImageProps> = ({ phrase }) => {
```

Comment:
I would consider moving this outside the Page component and just passing the Kids category as a prop :)

Reply:
@JuroUhlar I'm not sure I totally understand. So not to use useMemo and pass kidsCategory as getStaticProps? thanks

Reply:
Ah sorry for the late reply, I misspoke, I meant extracting the `PhraseImage` function compoenent outside the `KidsDictionaryPage`, I don't see the advantage of defining it in the render method here. But it's not a big deal, I merged the PR, feel free to tweak it in a separate one if you want :)